### DISCUSSION TITLE
asynchronous running js script

### DISCUSSION THREAD

**Discussion by ladiesman2127** [readium/kotlin-toolkit#523] [Q&A] [answered]
Hi. I am trying to display epub and run some js code that changes the color of words background. But the problem is that page renders after running script. Is that possible to change it. I've tried to run same html doc in browser, works fine, page renders first, then run script and modifies page
here's code
[index.txt](https://github.com/readium/kotlin-toolkit/files/15446697/index.txt)

**Comment by mickael-menu** [comment]
Have you tried wrapping your code in a `document.onload`?

**Comment by ladiesman2127** [comment]
I've just tried onload, it doesnt work at all, it even doesnt calls. Then i tried DOMContentLoaded. It works, but just as default (

**Comment by ladiesman2127** [comment, marked as answer]
Is there any way to do it?
